_api-reference/list/list-indices.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: List indices 
+parent: List API
+nav_order: 25
+has_children: false
+---
+
+# List indices
+**Introduced 2.18**
+{: .label .label-purple }
+
+The list indices operation provides the following index information in a paginated format: 
+
+- The amount of disk space used by the index. 
+- The number of shards contained in the index. 
+- The index's health status.
+
+## Path and HTTP methods
+
+```json
+GET _list/indices
+GET _list/indices/<index>
+```
+
+## Query parameters
+
+Parameter | Type | Description
+:--- | :--- | :---
+`bytes` | Byte size | Specifies the units for the byte size, for example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+`health` | String | Limits indexes based on their health status. Supported values are `green`, `yellow`, and `red`.
+`include_unloaded_segments` | Boolean | Whether to include information from segments not loaded into memory. Default is `false`.
+`cluster_manager_timeout` | Time | The amount of time to wait for a connection to the cluster manager node. Default is `30s`.
+`pri` | Boolean | Whether to return information only from the primary shards. Default is `false`.
+`time` | Time | Specifies the time units, for example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+`expand_wildcards` | Enum | Expands wildcard expressions to concrete indexes. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
+`next_token` | String | Fetches the next page of indexes. When `null`, only provides the first page of indexes. Default is `null`. 
+`size` | Integer | The maximum number of indexes to be displayed on a single page. The number of indexes on a single page of the response is not always equal to the specified `size`. Default is `500`. Maximum value is `5000`.
+`sort` | String | The order in which the indexes are displayed. If `desc`, then the most recently created indexes are displayed first. If `asc`, then the oldest indexes are displayed first. Default is `asc`.
+
+## Example requests
```

Comment:
@sumitasr: Happy to include the example if your team produces one. We can also add additional examples in a future PR.

Reply:
@Naarcha-AWS I'm adding the examples in the current PR itself, hope that's okay? Sorry for last minute additions. I'll be revising the PR addressing all the comments posted by @sumitasr.

Reply:
@sumitasr Since we are supporting requests and responses in different paragraphs, I have added instructions on passing the next_token. I think that should be sufficient for users to get hold of new APIs. Please check again.